newIDE/app/src/InstancesEditor/CompactInstancePropertiesEditor/CompactPropertiesSchema.js

```diff
@@ -504,12 +504,14 @@ export const makeSchema = ({
       {
         name: 'Rotation',
         type: 'row',
-        title: i18n._(t`Rotation`),
         preventWrap: true,
         removeSpacers: true,
-        children: getRotationXAndRotationYFields({ i18n }),
+        title: i18n._(t`Rotation`),
+        children: [
+          getRotationZField({ i18n }),
+          ...getRotationXAndRotationYFields({ i18n }),
```

Comment:
Ok so we validated the changes but I'm not sure why this works. It should display the 3 fields on a single row but on your screenshot, it's the Z field first on a whole row and then the X and Y together on a different row.